modules/administration-guide/pages/enabling-fuse-for-all-workspaces.adoc

```diff
@@ -7,6 +7,11 @@
 [id="enabling-fuse-overlayfs-for-all-workspaces"]
 = Enabling fuse-overlayfs for all workspaces
 
+For Podman 5.x, the `/home/user/.config` must be owned by the current user for Podman to function correctly.
+The `storage.conf` file for Podman is typically stored in this folder.
+This document explains how to configure the container entrypoint script for the workspace so that fuse-overlayfs is being used.
+The Universal Developer Image (UDI) already contains the necessary configuration by default.
+
 .Prerequisites
 
 * The xref:administration-guide:enabling-access-to-dev-fuse-for-openshift.adoc[] section has been completed. This is not required for OpenShift versions 4.15 and later.
```

Comment:
i'm wondering if it would make sense to reword the prerequisite since it's only necessary for older OCP versions.

maybe we should add FOR OCP VERSIONS 4.14 AND LOWER there?

Reply:
I've updated the wording:
```
For OpenShift versions 4.14 and lower, the xref:administration-guide:enabling-access-to-dev-fuse-for-openshift.adoc[]
section has been completed.
```